packages/blockchain/celo/src/lib/transactions/native/index.ts

```diff
@@ -0,0 +1,87 @@
+import { BroadcastFunction } from '@tatumio/shared-blockchain-abstract'
+import BigNumber from 'bignumber.js'
+import { EvmBasedBlockchain } from '@tatumio/shared-core'
+import { CeloWallet } from '@celo-tools/celo-ethers-wrapper'
+import Web3 from 'web3'
+import { celoUtils, CeloTransactionConfig, ChainTransferCeloBlockchain } from '../../utils/celo.utils'
+import { Erc20Token } from '@tatumio/shared-blockchain-evm-based'
+
+const transferSignedTransaction = async (
+  body: ChainTransferCeloBlockchain,
+  provider?: string,
+  testnet?: boolean,
+) => {
+  // TODO
+  // await validateBody(body, ChainTransferCeloBlockchain)
+
+  const { fromPrivateKey, signatureId, to, feeCurrency, amount, nonce } = body
+  const celoProvider = celoUtils.getProvider(provider)
+
+  if (to && feeCurrency && amount) {
+    const network = await celoProvider.ready
+    const feeCurrencyContractAddress = celoUtils.getFeeCurrency(feeCurrency, testnet)
+    const contract = new new Web3().eth.Contract(Erc20Token.abi as any, to.trim())
```

Comment:
Whats the difference between **native** and **erc20** txs? Can we use some method from erc20?

Reply:
The native transfer is related to https://tatum.io/apidoc.php#operation/CeloBlockchainTransfer which is just a ERC20/Celo/cUSD/ transfer between addresses. Since most of Celo is custom implemented and not shared I've gone the same route. Open to suggestions, since this especially is very new to me.